core/src/main/java/com/novoda/noplayer/model/TextCues.java

```diff
@@ -12,7 +12,7 @@ public TextCues(List<Cue> cues) {
         this.cues = cues;
     }
 
-    List<Cue> cues() {
+    public List<Cue> cues() {
```

Comment:
If the client needs to have those models then ok, but if it doesn't then these models become a contract with the clients